README.md

```diff
@@ -42,8 +42,10 @@ tft:
         plugins:
           - name: (15)
           - name: (15)
-kubeconfig: (16)
-kubeconfig_infra: (16)
+        secondary_network_nad: "(16)"
+    oci_bin: (17)
+kubeconfig: (18)
+kubeconfig_infra: (18)
```

Comment:
I think, `oci_bin` as a configuration parameter makes sense. But maybe it should be at the highest level of configuration (beside the `kubeconfig` entry). Do you expect to run tests with docker and some with podman, during the same run?

Reply:
maybe you could split this branch. A first commit that introduces `oci_bin`. That is pretty independent from the rest. It makes the individual commits smaller and easier to review. It also gives room for (multiple) commit messages that talk about the change.

Reply:
Sure! I've removed the `oci_bin` changes from this PR. Will raise a PR for those changes separately after this PR is merged